src/components/FeaturePanel/EditDialog/EditContent/FeatureEditSection/ParentsEditor.tsx

```diff
@@ -110,11 +119,24 @@ export const ParentsEditor = () => {
         id="panel1-header"
         onClick={() => setIsExpanded(!isExpanded)}
       >
-        <Stack direction="row" spacing={2} alignItems="center">
-          <Typography variant="button">
-            <SectionName />
-          </Typography>
-          <Chip size="small" label={parents.length} variant="outlined" />
+        <Stack
+          direction="row"
+          spacing={2}
+          alignItems="center"
+          justifyContent="space-between"
+          flex="1"
+        >
+          <Stack direction="row" spacing={2} alignItems="center">
+            <Typography variant="button">
+              <SectionName />
+            </Typography>
+            <Chip size="small" label={parents.length} variant="outlined" />
+          </Stack>
+          {isExpanded && (
+            <Button size="small" color="secondary" onClick={handleOpenAll}>
+              {t('editdialog.open_all')}
+            </Button>
```

Comment:
There can't be a `Button` inside `AccordionSummary`. The console was complaining 😸  Fixed in #1266

<img width="463" height="546" alt="Image" src="https://github.com/user-attachments/assets/78607987-517b-4dd5-bc81-b5b0685fde2d" />